packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -198,19 +201,5 @@ class Chrome {
     chromeConnection.close();
     _process?.kill();
     await _process?.exitCode;
```

Comment:
It doesn't look like stdout is drained anywhere.

Reply:
I didn't think that I needed to, since I don't listen to it. Shouldn't the conversion to a Future (for stderr) deal with draining?

Reply:
Yup, that'll work.